### PR TITLE
Python improvements

### PR DESCRIPTION
**Pull request by phuang1024**
* Improved Python IO instructions
* Instructions to run Python
* Formatting

### CODE REVIEW

**Review by dongliu0426 — CHANGES_REQUESTED** [review]

File: content/1_General/Running_Code_Locally.mdx [hunk 1]
```diff
@@ -130,11 +130,18 @@ features in Java.
...
 many features, such as syntax highlighting and automatic indentation, that one
-would normally expect from an IDE/text editor. However, feel free to use another
-editor or IDE if you want to.
+would normally expect from an IDE/text editor.
+
+### Linux
+
+You can use built-in terminal editors, like ``vim`` and ``nano``. You can also
+install editing software like VSCode. [For people who opened vim and can't exit]
+(https://stackoverflow.com/questions/11828270/how-do-i-exit-the-vim-editor)
```
Comment (dolphingarlic) [inline content/1_General/Running_Code_Locally.mdx:144]: Lmao nice

File: content/1_General/Running_Code_Locally.mdx [hunk 2]
```diff
@@ -709,12 +725,11 @@ used to programming in Python 2, that is OK too.
...
 Try both commands, and pay attention to the version that Python prints to
-determine if this is the case on your system. For example, running the following
-in Terminal on my computer produces:
+determine if this is the case on your system. For example:
 
 ```
-python --version # prints Python 2.7.13
-python3 --version # prints Python 3.8.1
+python --version   # Python 2.7.13
+python3 --version  # Python 3.8.1
```
Comment (dolphingarlic) [inline content/1_General/Running_Code_Locally.mdx:732]: Maybe keep the previous version to explicitly show that it prints the version?

File: content/1_General/Running_Code_Locally.mdx (hunk 1 above)
Comment (dongliu0426) [reply content/1_General/Running_Code_Locally.mdx:144]: 🙋 lol vim is so difficult to use

**Review by dolphingarlic — APPROVED** [review]
Just resolve the merge conflict and then I can merge

File: content/4_Gold/BFS.mdx [hunk 3]
```diff
@@ -69,6 +69,32 @@ System.out.println(q.peek()); // 3
 
 </JavaSection>
 
+<PySection>
+
+#### Python
+
+Python has a `queue` built-in module.
```
Comment (SansPapyrus683) [inline content/4_Gold/BFS.mdx:76]: queue's synchronized, so it's a bit slow
perhaps just tell them to use deque instead?

**Comment by thecodingwizard** [comment]
good enough, I'll merge. Feel free anyone to make another PR to address @SansPapyrus683 's comment

**Pull request merged**